code/modules/clothing/chameleon.dm

```diff
@@ -257,6 +257,53 @@
 	. = ..()
 	chameleon_action.emp_randomise(INFINITY)
 
+// Traitor item "Doppelganger Suit", behaves the same way as the reactive stealth armor, but has the added bonus of chameleon disguise.
+/obj/item/clothing/suit/chameleon/doppel
+	var/active = FALSE
+	resistance_flags = LAVA_PROOF | FIRE_PROOF | ACID_PROOF
+
+/obj/item/clothing/suit/chameleon/doppel/proc/doppel_cooldown(mob/user)
+	cooldown = FALSE
+
+/obj/item/clothing/suit/chameleon/doppel/proc/doppel_cloaking(mob/living/carbon/human/owner)
+	owner.alpha = initial(owner.alpha)
```

Comment:
Can you check if this breaks other abilities that make you near invisible? I'm not sure if those are constantly applied or not

Reply:
tested it with changelings’, the clone is invisible if you’re invisible, and when the ability ends, the user is visible. tbh i’m considering this is fine, given that this and the genetic power are the only things that turn the user invisible. (and also to activate the ability you gotta hit the wearer)